Update rdp-shortpath.md (#126926)
Azure/azure-content: articles/virtual-desktop/rdp-shortpath.md

The following sections contain the source, destination and protocol requirements for your session hosts and client devices that must be allowed for RDP Shortpath to work.

> [!NOTE]

> > Starting **June 15**, Microsoft will begin rolling out a new TURN relay IP range, **51.5.0.0/16**, across **40 Azure regions**. This new range is **dedicated exclusively to Azure Virtual Desktop (AVD) and Windows 365 (W365)**, marking a transition away from the previously shared **20.202.0.0/16** subnet used by Azure Communication Services. The upgrade is designed to enhance RDP Shortpath for Public Networks (via TURN/Relay), delivering **faster, more reliable connectivity** for AVD and W365 users.

Reviewer: ```suggestion
> Starting June 15, Microsoft will begin rolling out a new TURN relay IP range, `51.5.0.0/16`, across 40 Azure regions. This new range is dedicated exclusively to Azure Virtual Desktop and Windows 365, marking a transition away from the previously shared 20.202.0.0/16 subnet used by Azure Communication Services. The upgrade is designed to enhance RDP Shortpath for Public Networks (via TURN/Relay), delivering faster, more reliable connectivity for users.
```

| STUN infrastructure/TURN relay | Client network | Any | `20.202.0.0/16` | 3478 | UDP | Allow |
| TURN relay | Client network | Any | `51.5.0.0/16` | 3478 | UDP | Allow |

> [!NOTE]

> > From June 15th, the traffic will progressively __be redirected__ from the current Azure Communication Service (ACS) TURN Relay range (**`20.202.0.0/16`**) to a newly designated subnet (**`51.5.0.0/16`**). While this shift is designed to be seamless, it’s essential that customers **preemptively configure bypass rules** for the new range to maintain uninterrupted service. With both IP ranges properly bypassed, end users will not experience any connectivity issues

Reviewer: ```suggestion
> From June 15th, the traffic will progressively be redirected from the current Azure Communication Service (ACS) TURN Relay range (`20.202.0.0/16`) to the newly designated subnet `51.5.0.0/16`. While this shift is designed to be seamless, it’s essential that customers preemptively configure bypass rules for the new range to maintain uninterrupted service. With both IP ranges properly bypassed, end users shouldn't experience any connectivity issues.
```
